Add component size for icon buttons

Vestad opened the pull request:
## 💡 Hvorfor?

Større ikonknapper som kan brukes på automat. 

## 🔧 Hvordan?

<!-- Forklar hvordan endringene er implementert -->

## 🧩 Type endring

<!-- Kryss av det som passer -->

- [ ] 🐞 Feilretting
- [x] 🚀 Ny funksjonalitet
- [ ] 💥 Breaking change (krever kodeendringer hos brukere)
- [ ] 📝 Dokumentasjonsoppdatering
- [ ] 🧹 Refaktorering (ingen funksjonelle endringer)
- [ ] ⚡️ Ytelsesforbedring
- [ ] 🏗️ Bygg-/CI-endring

## 🖼️ Skjermbilder

<img width="1191" height="693" alt="Skjermbilde 2025-12-10 kl  15 22 00" src="https://github.com/user-attachments/assets/078a0146-affa-4880-8eaa-c96135b09bb7" />

## 💬 Tilleggsnotater

<!-- Andre tanker, bekymringer eller relevante lenker -->

## 💣 Breaking changes (om aktuelt)

<!-- Beskriv hva som brytes og hvordan man migrerer -->

<!-- Fjern seksjonen om ikke relevant -->

## ✅ Sjekkliste

<!-- Kryss av når punktene er oppfylt -->

- [ ] Navnestandarder følges
- [ ] Kode og Figma reflekterer hverandre
- [ ] Dokumentasjonen er oppdatert (hvis aktuelt)
- [ ] Ingen ubrukte imports / varsler / console.logs
- [ ] Funker og ser ut som forventet i pakker som avhenger av denne

## 🧪 Testing

<!-- Hvordan er endringen testet og verifisert? -->

- [ ] Testet med skjermleser
- [ ] Testet i Safari og Firefox
- [ ] Testet i dokumentasjonsiden

asewilhelmsen committed:
feat(button/button): reduce css specificity for all button classes to 0,1,0

BREAKING CHANGE: New specificity for classes could lead to visual changes if you have overridden
button classes

magnusrand committed:
chore(release): publish

 - @entur/code-playground@2.3.10-beta-automat.0
 - @entur/documentation@6.13.1-beta-automat.0
 - studio-linje@1.8.1-beta-automat.0
 - @entur/a11y@0.2.104-beta-automat.0
 - @entur/alert@0.18.1-beta-automat.0
 - @entur/button@3.4.1-beta-automat.0
 - @entur/chip@0.10.1-beta-automat.0
 - @entur/datepicker@11.5.1-beta-automat.0
 - @entur/dropdown@8.0.3-beta-automat.0
 - @entur/expand@3.7.1-beta-automat.0
 - @entur/fileupload@0.5.1-beta-automat.0
 - @entur/form@9.2.1-beta-automat.0
 - @entur/grid@0.3.70-beta-automat.0
 - @entur/icons@8.3.2-beta-automat.0
 - @entur/layout@3.3.1-beta-automat.0
 - @entur/loader@0.6.1-beta-automat.0
 - @entur/menu@5.3.1-beta-automat.0
 - @entur/modal@1.8.1-beta-automat.0
 - @entur/tab@0.6.1-beta-automat.0
 - @entur/table@4.10.1-beta-automat.0
 - @entur/tokens@3.21.1-beta-automat.0
 - @entur/tooltip@5.3.1-beta-automat.0
 - @entur/travel@6.5.1-beta-automat.0
 - @entur/typography@2.1.1-beta-automat.0
 - @entur/utils@0.12.6-beta-automat.0

magnusrand committed:
feat(button/icon button): add view-mode support for IconButton

Support for standard, compact and display sizes.

BREAKING CHANGE: New values for sizes may cause visual changes.

ETU-65808

magnusrand committed:
feat(button/button): change minimum width for button to the same as button height

This change makes button more customisable, allowing more compact usage.

BREAKING CHANGE: Buttons with short text are likely to shrink in size. Set `min-width: 9.5rem;` to
preserve old behaviour.

magnusrand committed:
feat(button/button): add support for dynamic typography size

Use typographyTokens from @entur/typography if available.

magnusrand committed:
chore: update stylelint config

Allow for BEM naming and compound selectors

magnusrand committed:
feat(button/button): add support for view mode in button

Button component will adapt based on data-view-mode being set to standard, compact or display.

BREAKING CHANGE: The new architecture could include small visual changes.

magnusrand committed:
feat(button/button): update border radius for button components

This change intends to give the buttons a more friendly and modern look.

magnusrand committed:
feat(button/button): update border radius for button components

This change intends to give the buttons a more friendly and modern look.

magnusrand committed:
fix(button/button): button overflow now results in elipsis instead of visual overflow

magnusrand committed:
chore: update yarn.lock

magnusrand committed:
Merge pull request #205 from entur/ETU-64136-legge-inn-component-size-i-button

feat(button): add manual component size file for buttons

magnusrand reviewed (changes requested):
De nye ikonknappene har ikke likt standarddesign som de eksisterende. Dette kan være problematisk; det må avklares med design at dette faktisk er intensjonen. I tillegg er standarddesignet heller ikke kvadratisk selv om alle andre størrelser er det. Jeg antar at dette også ikke er gjort med hensikt – avklares med design.

<img width="808" height="513" alt="Image" src="https://github.com/user-attachments/assets/0c04aed0-f2db-4b4f-a1d3-c27a2cf9a7e7" />

magnusrand committed:
feat(button/icon button): change specificity for all icon button classes to 0,1,0

This makes overriding and importing the classes more predictable

BREAKING CHANGE: Specificity changes could cause visual changes if you override the classes
